app/src/main/java/com/plaid/linksample/MainActivity.kt

```diff
@@ -74,14 +78,22 @@ class MainActivity : AppCompatActivity() {
    * [parameter reference](https://plaid.com/docs/link/android/#parameter-reference).
    */
   private fun openLink() {
-    Plaid.openLink(
-      activity = this,
-      linkConfiguration = LinkConfiguration(
-        clientName = "Link demo",
-        products = listOf(PlaidProduct.TRANSACTIONS)
-      ),
-      requestCode = LINK_REQUEST_CODE
-    )
+    // We create an item-add-token in order to authenticate item creation.
+    linkSampleApi.getItemAddToken()
```

Comment:
Since this is the only place we are using Rx maybe we should just use retrofit's `Call` to reduce the dependencies and make things clearer for developers who don't know Rx

Reply:
So I tried swapping it back to `Call` and it actually looked even more confusing. I think Retrofit `Calls` are so rarely used (vs Rx and coroutines) that we might gain more clarity with using `Rx`.